h/views/admin/groups.py

```diff
@@ -217,7 +223,9 @@ def _update_appstruct(self):
                 "group_type": group.type,
                 "name": group.name,
                 "members": [m.username for m in group.members],
-                "organization": group.organization.pubid,
+                "organization": (
+                    group.organization.pubid if group.organization else None
+                ),
```

Comment:
This happens when loading the group edit page. If the group has no organization then `group.organization` will be `None` in which case we now set `appstruct["organization"]` to `None` which causes the form to render with `-- None --` selected for the organization, rather than crashing during page load. (This was the original crash from https://github.com/hypothesis/h/issues/9009, but fixing this page load crash then reveals the various other crashes fixed above that happen when trying to actually submit the form).